scaffold/github/workflows/ComposerLockDiff.yml

```diff
@@ -43,7 +43,7 @@ jobs:
             -H "Authorization: Bearer  ${{ secrets.GITHUB_TOKEN }}" \
             https://api.github.com/repos/${{ github.repository }}/pulls/$DRAINPIPE_PR_NUMBER | jq '. | {body}' > pull_request.json
           sed -i 's/<!--Composer Lock Diff-->.*<!--\/Composer Lock Diff-->//g' pull_request.json
-          DESCRIPTION=$(cat pull_request.json | jq -r '.body' | sed -z 's/\n/\\n/g')
+          DESCRIPTION=$(cat pull_request.json | jq -r '.body' | sed -z 's/["\n\r]/\\n/g')
```

Comment:
Confirming that this has fixed a couple projects where we were originally getting errors from curl.